my-app/src/components/Docs/FileUploader.js

```diff
@@ -0,0 +1,60 @@
+import FileUploader from "react-firebase-file-uploader";
+import React, {Component} from 'react';
+import firebase from 'firebase';
+
+
+
+class file extends Component {
+
+  state = {
+    image: '',
+    imageURL: '',
+    progress: 0
+  }
+
+
+  handleUploadStart = () => {
+
+    this.setState({
+      progress: 0
+    })
+  }
+
+  handleUploadSuccess = filename => {
+    this.setState ({
+      image: filename,
+      progress: 100
+    })
+
+    firebase.storage().ref('avatars').child(filename).getDownloadURL()
+    .then(url => this.setState({
```

Comment:
here you should bubble up event (send this url to parent) - so you could save it to document as new field